deployment/deploy/3000-deploy-VestingRegistry.js

```diff
@@ -0,0 +1,25 @@
+const path = require("path");
+const { getContractNameFromScriptFileName } = require("../helpers/utils");
+const { sendWithMultisig } = require("../helpers/helpers");
+const { deployWithCustomProxy } = require("../helpers/helpers");
+const col = require("cli-color");
+//const deploymentName = getContractNameFromScriptFileName(path.basename(__filename));
+const func = async function (hre) {
+    const {
+        deployments: { deploy, get, log },
+        getNamedAccounts,
+        ethers,
+    } = hre;
+    const { deployer } = await getNamedAccounts();
+    log(col.bgYellow("Deploying VestingRegistry..."));
+    await deployWithCustomProxy(
+        deployer,
+        "VestingRegistryLogic",
+        "VestingRegistryProxy",
+        "",
+        "VestingRegistry_Proxy",
```

Comment:
because there is already VestingRegistryProxy deployment data saved, either rename the deployment instance:  
```suggestion
        "VestingRegistryProxy",
```
or rename the deployment file on both mainnet and testnet